presto-tests/src/test/java/io/prestosql/tests/TestWorkProcessorPipelineQueries.java

```diff
@@ -46,6 +46,7 @@ protected TestWorkProcessorPipelineQueries()
     public void testTopN()
     {
         assertLazyQuery("SELECT * FROM orders ORDER BY totalprice LIMIT 10");
+        assertLazyQuery("SELECT * FROM orders WHERE orderkey >= 0 ORDER BY totalprice LIMIT 10");
```

Comment:
IIRC all order keys are positive, so one day this filter could be eliminated by table constraints.  Maybe pick a value like `10`?